compiler/cpp/src/thrift/generate/t_kotlin_generator.cc

```diff
@@ -1601,43 +1599,43 @@ void t_kotlin_generator::generate_service_processor(t_service* tservice) {
   out << autogen_comment() << warning_surpressions() << kotlin_package();
   auto service_imports = {"import kotlinx.coroutines.future.future"};
   for (auto service_import : service_imports) {
-    out << service_import << endl;
+    out << service_import << '\n';
   }
-  out << endl;
+  out << '\n';
 
   generate_docstring_comment(out, "/**\n", " * ",
                              "server implementation for [" + tservice->get_name() + "]", " */\n");
-  indent(out) << "class " << tservice->get_name() << "Processor(" << endl;
+  indent(out) << "class " << tservice->get_name() << "Processor(" << '\n';
   indent_up();
-  indent(out) << "handler: " << tservice->get_name() << "," << endl;
-  indent(out) << "private val scope: kotlinx.coroutines.CoroutineScope," << endl;
+  indent(out) << "handler: " << tservice->get_name() << "," << '\n';
+  indent(out) << "private val scope: kotlinx.coroutines.CoroutineScope," << '\n';
   indent(out) << "private val processMap: kotlin.collections.Map<kotlin.String, "
                  "org.apache.thrift.AsyncProcessFunction<"
               << tservice->get_name()
               << ", out org.apache.thrift.TBase<*, "
-                 "*>, out kotlin.Any, out org.apache.thrift.TBase<*, *>>> = mapOf("
-              << endl;
+                 "*>, out kotlin.Any>> = mapOf("
+              << '\n';
   indent_up();
```

Comment:
Probably merge failure ... that came in with 2826098 THRIFT-5762 Fix spotless errors
 `, out org.apache.thrift.TBase<*, *>`